backend/eventloop/eventlooptypes/types.go

```diff
@@ -13,7 +13,7 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	operation "github.com/redhat-appstudio/managed-gitops/backend-shared/apis/managed-gitops/v1alpha1"
+	managedgitopsbackendsharedv1alpha1 "github.com/redhat-appstudio/managed-gitops/backend-shared/apis/managed-gitops/v1alpha1"
```

Comment:
this is too long and as a result later when you use it the lines tend to get longer as well. You can just pick another shorter name for this (eg. `shared`) instead of `managedgitopsbackendsharedv1alpha1`